### ISSUE TITLE
Error No such file or directory: 'screen'

### ISSUE THREAD

**Issue by LeducH** [ranger/ranger#2580]
#### Runtime Environment

- Operating system and version: (BTW) Arch Linux x86_64 
- Terminal emulator and version:  alacritty  0.10.1-1
- Python version: Python 3.9.2
- Ranger version/commit: in Traceback
- Locale: in Traceback
- WM: bspwm

#### Current Behavior
Ranger does not start

#### Expected Behavior
Ranger starts 

#### Context

I want to open ranger 

#### Possible Solutions

fixing screen dependency

#### Steps to reproduce
1. type 'ranger' in terminal press enter
2. 
3. 

#### Traceback

```
ranger version: ranger 1.9.3
Python version: 3.9.2 (default, Feb 20 2021, 18:40:11) [GCC 10.2.0]
Locale: en_IE.UTF-8

Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/ranger/core/main.py", line 171, in main
    fm.initialize()
  File "/usr/lib/python3.9/site-packages/ranger/core/fm.py", line 132, in initialize
    self.ui.initialize()
  File "/usr/lib/python3.9/site-packages/ranger/gui/ui.py", line 127, in initialize
    self.handle_multiplexer()
  File "/usr/lib/python3.9/site-packages/ranger/gui/ui.py", line 499, in handle_multiplexer
    self._screen_title = check_output(
  File "/usr/lib/python3.9/site-packages/ranger/ext/spawn.py", line 35, in check_output
    process = Popen(popenargs, stderr=fd_devnull, **kwargs)
  File "/usr/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.9/subprocess.py", line 1823, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'screen'

```

no still not working 

```
OS: Arch Linux x86_64
Host: 20KH0035UK ThinkPad X1 Carbon 6th
Kernel: 5.16.11-arch1-1
Uptime: 14 mins
Packages: 1554 (pacman)
Shell: zsh 5.8.1
Resolution: 1920x1080
WM: bspwm
Theme: Breeze-Dark [GTK2/3]
Icons: Papirus-Dark [GTK2/3]
Terminal: alacritty
Terminal Font: Fira Code
CPU: Intel i5-8250U (8) @ 3.400GHz
GPU: Intel UHD Graphics 620
Memory: 2332MiB / 7483MiB
```

```
ranger version: ranger 1.9.3
Python version: 3.9.2 (default, Feb 20 2021, 18:40:11) [GCC 10.2.0]
Locale: en_IE.UTF-8

Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/ranger/core/main.py", line 171, in main
    fm.initialize()
  File "/usr/lib/python3.9/site-packages/ranger/core/fm.py", line 132, in initialize
    self.ui.initialize()
  File "/usr/lib/python3.9/site-packages/ranger/gui/ui.py", line 127, in initialize
    self.handle_multiplexer()
  File "/usr/lib/python3.9/site-packages/ranger/gui/ui.py", line 499, in handle_multiplexer
    self._screen_title = check_output(
  File "/usr/lib/python3.9/site-packages/ranger/ext/spawn.py", line 35, in check_output
    process = Popen(popenargs, stderr=fd_devnull, **kwargs)
  File "/usr/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.9/subprocess.py", line 1823, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'screen'

```

```
% pacman -Q --info ranger
Name            : ranger
Version         : 1.9.3-4
Description     : Simple, vim-like file manager
Architecture    : any
URL             : https://ranger.github.io
Licenses        : GPL
Groups          : None
Provides        : None
Depends On      : python
Optional Deps   : atool: for previews of archives [installed]
                  elinks: for previews of html pages
                  ffmpegthumbnailer: for video previews [installed]
                  highlight: for syntax highlighting of code [installed]
                  libcaca: for ASCII-art image previews [installed]
                  lynx: for previews of html pages [installed]
                  mediainfo: for viewing information about media files [installed]
                  odt2txt: for OpenDocument texts [installed]
                  perl-image-exiftool: for viewing information about media files
                  poppler: for pdf previews [installed]
                  python-chardet: in case of encoding detection problems [installed]
                  sudo: to use the "run as root"-feature [installed]
                  transmission-cli: for viewing bittorrent information
                  ueberzug: for previews of images
                  w3m: for previews of images and html pages [installed]
Required By     : None
Optional For    : None
Conflicts With  : None
Replaces        : None
Installed Size  : 1861.82 KiB
Packager        : Felix Yan <felixonmars@archlinux.org>
Build Date      : Fri 03 Dec 2021 02:00:52
Install Date    : Thu 23 Dec 2021 19:14:54
Install Reason  : Explicitly installed
Install Script  : No
Validated By    : Signature
```

**Comment by markus-bauer** [comment]
Is this a duplicate of: https://github.com/ranger/ranger/issues/1805?

Possibly fixed here: https://github.com/ranger/ranger/commit/1cdcce0a44d031be13009a70e25cc4cf75789eac.
Can you try the master branch and see if it works?

**Comment by LeducH** [comment]
> Yes, I suspect this has been fixed on master. To test you could clone the repository and then run `python path/to/repo/ranger.py`.

Is it in released yet since the version is still von `1.9.3` 
and the package in arch reflects that https://archlinux.org/packages/community/any/ranger/

**Comment by toonn** [comment]
Not released yet but gonna close as fixed. On Arch you could run `ranger-git` from the AUR to workaround this.

Fixed in #1852.
